harvest/api/_base.py

```diff
@@ -878,15 +891,15 @@ def timeout(self):
             self.flush()
 
     def flush(self):
-        # For missing data, repeat the existing one
+        # For missing data, return a OHLC with all zeroes.
         self.block_lock.acquire()
         for n in self.needed:
-            data = (
-                self.trader.storage.load(n, self.interval[n]["interval"])
-                .iloc[[-1]]
-                .copy()
+            data = pd.DataFrame(
+                {"open": 0, "high": 0, "low": 0, "close": 0, "volume": 0},
```

Comment:
replace  with all zeroes because it does not rely on needed storage and it can let users handle this case.